dockerhub-prune-tags.py

```diff
@@ -71,33 +71,37 @@ def authstr(user, passwd):
     from base64 import encodestring
     return encodestring('%s:%s' % (user,passwd)).replace('\n', '')
 
+
+## unsure about urllib request and response. It was originally
+# 'urllib2.request(url)' and 'urllib2.urlopen(req)'
```

Comment:
Well, it was

```suggestion
# 'urllib2.Request(url)' and 'urllib2.urlopen(req)'
```

I think the "upgrade" from `urllib2` -> `urllib.request` is correct for these, but you need to `import urllib.request` explicitly.

I would appreciate you testing these to the point that you are no longer "unsure" about it though, and then removing these two comment lines!  :)